server/data_provider_canvas_esb.rb

```diff
@@ -76,22 +85,29 @@ def initConfigureCanvasESBProvider(config_hash)
 
   def canvasAPICalendarEventsURL(uniqname, canvas_courses)
 
-    ## calculate a 15 day range around now for the assignment search (last week, today, next week)
+    ## calculate a multi day range around now for the assignment search (last week, today, next week).
+    ## Can override defaults (above) in the studentdashboard.yml file.
```

Comment:
There should be better documentation.  The app looks for the studentdashboard.yml but we configure the file by creating a symbolic link to the real studentdashboard.yml file.  So there are lots of separate configuration files for different purposes in source control.